stl/src/filesystem.cpp

```diff
@@ -750,7 +758,7 @@ __std_win_error __stdcall __std_fs_get_file_id(__std_fs_file_id* const _Id, cons
 }
 
 [[nodiscard]] __std_win_error __stdcall __std_fs_resize_file(
-    const wchar_t* const _Target, const uintmax_t _New_size) noexcept {
+    _In_z_ const wchar_t* const _Target, const uintmax_t _New_size) noexcept {

```

Comment:
I observe that this isn't marking `const uintmax_t _New_size` as `_In_` - but that has no effect for value parameters anyways. No change requested.